backend/items/api.py

```
@@ -266,6 +366,167 @@ class CheckupTypeSchema(Schema):
+class LocationMoveSchema(Schema):
+    parent_id: Optional[UUID] = None
+
+
+class LocationTreeNode(Schema):
+    id: str
+    slug: str
+    display_name: str
+    full_path: str
+    level: int
+    parent_id: Optional[str] = None
+    children: List['LocationTreeNode'] = []
```

Ruff (0.15.2)

[warning] Mutable default value for class attribute

(RUF012)